src/Altinn.Apps/AppTemplates/AspNet/Altinn.App.Api/Controllers/StatelessDataController.cs

```diff
@@ -25,40 +36,51 @@ public class StatelessDataController : ControllerBase
         private readonly IAltinnApp _altinnApp;
         private readonly IAppResources _appResourcesService;
         private readonly IPrefill _prefillService;
+        private readonly IRegister _registerService;
+        private readonly IPDP _pdp;
+        private readonly IAltinnAppContext _altinnAppContext;
 
         private const long REQUEST_SIZE_LIMIT = 2000 * 1024 * 1024;
 
+        private const string Partyheader = "party";
+        private const string PartyPrefix = "partyid:";
+        private const string PersonPrefix = "person:";
+        private const string OrgPrefix = "org:";
+
         /// <summary>
         /// The stateless data controller is responsible for creating and updating stateles data elements.
         /// </summary>
-        /// <param name="logger">The logger</param>
-        /// <param name="altinnApp">The app logic for current application</param>
-        /// <param name="appResourcesService">The apps resource service</param>
-        /// <param name="prefillService">A service with prefill related logic.</param>
         public StatelessDataController(
             ILogger<DataController> logger,
             IAltinnApp altinnApp,
             IAppResources appResourcesService,
-            IPrefill prefillService)
+            IPrefill prefillService,
+            IRegister registerService,
+            IPDP pdp,
+            IAltinnAppContext altinnAppContext)
         {
             _logger = logger;
-
             _altinnApp = altinnApp;
             _appResourcesService = appResourcesService;
             _prefillService = prefillService;
+            _registerService = registerService;
+            _pdp = pdp;
+            _altinnAppContext = altinnAppContext;
         }
 
         /// <summary>
         /// Create a new data object of the defined data type
         /// </summary>
+        /// <param name="org">unique identfier of the organisation responsible for the app</param>
+        /// <param name="app">application identifier which is unique within an organisation</param>
         /// <param name="dataType">The data type id</param>
         /// <returns>Return a new instance of the data object including prefill and initial calculations</returns>
         [Authorize]
         [HttpPost]
         [DisableFormValueModelBinding]
         [RequestSizeLimit(REQUEST_SIZE_LIMIT)]
         [ProducesResponseType(typeof(DataElement), 200)]
-        public async Task<ActionResult> Post([FromQuery] string dataType)
+        public async Task<ActionResult> Post(string org, string app, [FromQuery] string dataType)
```

Comment:
Could include the  [FromRoute] tag for org, app to make it easier to understand where they are retrieved from

Reply:
agree